src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorMessages } from '../../error.messages';
+
+export const validateGithubHandles = (value: string) => {
```

Comment:
more semantic param name would be `handles: string`, could be multiple so should be plural